src/Microsoft.Data/Microsoft.Data.DataFrame.csproj

```diff
@@ -0,0 +1,114 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Buffers" Version="4.5.0" />
+    <PackageReference Include="System.Memory" Version="4.5.1" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.1" />
+    <PackageReference Include="System.Text.Encoding" Version="4.3.0" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <None Update="BaseColumn.BinaryOperations.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>BaseColumn.BinaryOperations.cs</LastGenOutput>
+    </None>
+    <None Update="BaseColumn.BinaryOperators.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>BaseColumn.BinaryOperators.cs</LastGenOutput>
+    </None>
+    <None Update="DataFrameBinaryOperations.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>DataFrameBinaryOperations.cs</LastGenOutput>
+    </None>
+    <None Update="DataFrameBinaryOperators.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>DataFrameBinaryOperators.cs</LastGenOutput>
+    </None>
+    <None Update="DataFrameColumnArithmeticTemplate.ttinclude">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>DataFrameColumnArithmeticTemplate.cs</LastGenOutput>
+    </None>
+    <None Update="PrimitiveColumn.BinaryOperations.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>PrimitiveColumn.BinaryOperations.cs</LastGenOutput>
+    </None>
+    <None Update="PrimitiveColumnArithmetic.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>PrimitiveColumnArithmetic.cs</LastGenOutput>
+    </None>
+    <None Update="PrimitiveColumnContainer.BinaryOperations.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>PrimitiveColumnContainer.BinaryOperations.cs</LastGenOutput>
+    </None>
+    <None Update="PrimitiveDataFrameColumnArithmetic.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>PrimitiveDataFrameColumnArithmetic.cs</LastGenOutput>
+    </None>
+    <None Update="PrimitiveDataFrameColumnContainer.BinaryOperations.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>PrimitiveDataFrameColumnContainer.BinaryOperations.cs</LastGenOutput>
+    </None>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Update="BaseColumn.BinaryOperations.cs">
```

Comment:
I don't see this file in your PR. Are you sure all of these are still necessary?